### PR TITLE
quest 1 and 2

### PR DESCRIPTION
**Pull request by nrhodes**

### CODE REVIEW

**Comment by nrhodes** [comment]
Current dependencies on/for this PR:
* main
  * **PR #5** <a href="https://app.graphite.dev/github/pr/nrhodes/hw-generation/5" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/nrhodes/hw-generation/5?utm_source=stack-comment).

File: data.py [hunk 1]
```diff
@@ -20,14 +22,28 @@ def collate_fn(samples):
         strokes_mask[len(strokes[i]):,i] = 0
 
     return all_texts, texts_mask, all_strokes, strokes_mask
-    
+
+def infinite_dl(dl):
```
Comment (sotelo) [inline data.py:26]: Looks great!

File: data.py [hunk 2]
```diff
@@ -20,14 +22,28 @@ def collate_fn(samples):
...
+    while True:
+        for batch in iter(dl):
+            yield batch
+
+def align_stroke(stroke):
```
Comment (sotelo) [inline data.py:31]: This is an interesting solution to the task. I saw the samples and it seems like it works well. However, one issue with this way of doing things is that we're modifying the empiric relation between x and y coordinates.

How could we do this in a way that would preserve the relation between x and y untouched?

Comment (nrhodes) [reply data.py:31]: Good point. I guess we could use the avg to calculate an angle of rotation, and then update each point.

**Pull request closed without merging**